alpaca_trade_api/polygon/entity.py

```diff
@@ -7,6 +7,11 @@
 class Entity(object):
     def __init__(self, raw):
         self._raw = raw
+        if 'from' in self._raw:
+            # can't use python keyword 'from'. if the api returns it,
+            # we switch it to _from, which is usable for the users.
+            self._raw['_from'] = self._raw['from']
+            del self._raw['from']
```

Comment:
you don't need to delete it do you?

Reply:
ok I will not delete it.
doing it on getattr will not work since the error is invoked by the python ide
you can't write something like this:
```py
data = live_api.polygon.daily_open_close('AAPL', '2020-05-29')
data.from
```